File: Wechat_chatroom_helper_android/app/src/main/java/com/zdy/project/wechat_chatroom_helper/ui/MyApplication.java
package com.zdy.project.wechat_chatroom_helper.ui;

import android.app.Application;

import com.tencent.bugly.Bugly;
import com.tencent.bugly.crashreport.CrashReport;


/**
 * Created by zhudo on 2017/7/25.
 */

public class MyApplication extends Application {


    private static MyApplication INSTANCE;

    @Override
    public void onCreate() {
        super.onCreate();

        Bugly.init(getApplicationContext(), "ed7bb0e103", false);


        INSTANCE = this;
    }

    public static MyApplication getINSTANCE() {
        return INSTANCE;
    }
}

File: Wechat_chatroom_helper_android/app/src/main/java/com/zdy/project/wechat_chatroom_helper/kt/ui/MyApplication.kt
package com.zdy.project.wechat_chatroom_helper.kt.ui

import android.app.Application
import com.tencent.bugly.Bugly
import com.zdy.project.wechat_chatroom_helper.ui.MyApplication

/**
 * Created by Mr.Zdy on 2017/10/19.
 */
class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        Bugly.init(applicationContext, "ed7bb0e103", false)

        instance = this

    }

    companion object {
        private var instance: com.zdy.project.wechat_chatroom_helper.kt.ui.MyApplication? = null

        fun get(): com.zdy.project.wechat_chatroom_helper.kt.ui.MyApplication {
            return instance!!
        }

    }
}
